lib/v2/screens/profile_screens/guardians/guardians_tabs/guardians_screen.dart

```diff
@@ -37,7 +39,12 @@ class GuardiansScreen extends StatelessWidget {
                 SnackBarInfo(pageCommand.message, ScaffoldMessenger.of(context)).show();
               } else if(pageCommand is ShowMessage) {
                 SnackBarInfo(pageCommand.message, ScaffoldMessenger.of(context)).show();
+              } else if (pageCommand is ShowOnboardingGuardianSingleAction){
+                _showOnboardingGuardianDialogSingleAction( pageCommand ,context);
+              }else if (pageCommand is ShowOnboardingGuardianDoubleAction){
+                _showOnboardingGuardianDialogDoubleAction(pageCommand, context);
               }
+
             },
             child: BlocBuilder<GuardiansBloc, GuardiansState>(builder: (context, state) {
```

Comment:
@gguijarro-c-chwy Should we set to use only `BlocConsumer` instead of `BlocBuilder` + `BlocListener`?

Thus we avoid the nesting of one more widget.

Reply:
yes